integration/spring-dsl-integration/docker/docker-compose-app.yml

```
@@ -0,0 +1,13 @@
+name: spring-dsl-integration-app
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)